scripts/docker/clean-logs.sh

```
@@ -35,7 +35,7 @@ while true; do
     -type f -mtime +"${RETENTION}" -name '*.log' -print0 | \
     xargs -0 rm -f
 
-  find "${DIRECTORY}"/logs -type d -empty -delete
+  trap find "${DIRECTORY}"/logs -type d -empty -delete ERR
```

This fix is simple and works but the disadvantage of it that every ERR is except here.
There is other solution that will except the specific file not found exception:

```bash
error_file=$(mktemp)
find "${DIRECTORY}"/logs -type d -empty -delete

if grep -q "No such file or directory" "$error_file"; then
    cat "$error_file"
    echo "Info: if you run with hight available scheduler probably the other scheduler already delete it"
fi

rm "$error_file"
```

---

Why not just `find "${DIRECTORY}"/logs -type d -empty -delete || true` ? That will ignore any errors. I think it should be pretty enough.

---

Yep. you are right. :)